### PR TITLE
feat(shop) connect local storage to redux store

### PR DESCRIPTION
**Pull request by m-spilman**

### CODE REVIEW

File: src/store/products/reducers.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import actions from './action-types';
+let products = JSON.parse(localStorage.getItem('products'));
```
Comment (BarretteBanner) [inline src/store/products/reducers.js:3]: leave a blank line between imports and your variables

**Review by BarretteBanner — APPROVED** [review]

File: src/components/add-products/add-products.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import React, { useState } from 'react';
 import { useDispatch } from 'react-redux';
 
-import { addProductAction } from '../../store/add-product/action';
 import BuildForm from '../form-builder';
 import ProductCard from './product-card';
+import { productAction } from '../../store/products/action';
```
Comment (ahaly92) [inline src/components/add-products/add-products.js]: nit picky but rename to productActions

File: src/store/products/reducers.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import actions from './action-types';
+
+let products = JSON.parse(localStorage.getItem('products'));
+const initialState = products ? { loading: true, products } : {};
```
Comment (ahaly92) [inline src/store/products/reducers.js]: initial state should have loading set to false

File: src/utils/fake-backend.js [hunk 4]
```diff
@@ -31,6 +31,7 @@ export function configureFakeBackend() {
 
       function addProduct() {
         const product = body;
```
Comment (ahaly92) [inline src/utils/fake-backend.js]: i would remove this assignment and just use body in the next line to pull out the id

**Review by ahaly92 — APPROVED** [review]
Some minor things here and there but overall this looks great!

**Pull request merged**